edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookServiceHibernateImpl.java

```diff
@@ -2257,6 +2258,120 @@ public Object doInHibernate(Session session) throws HibernateException {
 	  }
   }
 
+	@Override
+	public void saveExcludedStatusForStudentGrade(String gradebookUid, Long gradableObjectId, String studentUid, Boolean excludedStatus) {
+		if (gradebookUid == null || gradableObjectId == null || studentUid == null) {
+			throw new IllegalArgumentException("Null gradebookUid or gradableObjectId or studentUid passed to saveExcludedStatusForStudentGrade");
+		}
+
+		GradeDefinition gradeDef = new GradeDefinition();
+		gradeDef.setStudentUid(studentUid);
+		gradeDef.setGradeExcluded(excludedStatus);
+
+		List<GradeDefinition> gradeDefList = new ArrayList<GradeDefinition>();
+		gradeDefList.add(gradeDef);
+
+		saveGradeExcluded(gradebookUid, gradableObjectId, gradeDefList);
+	}
+
+	@Override
+	public void saveGradeExcluded(final String gradebookUid, final Long gradableObjectId, List<GradeDefinition> gradeDefList) {
```

Comment:
Can you rename this method? I can see the method chain from method above but a method overload would be a better approach (ie name them the same thing)